Sources/EmbraceCore/FileSystem/EmbraceFileSystem.swift

```diff
@@ -53,7 +53,8 @@ public struct EmbraceFileSystem {
     /// ```
     /// - Parameters:
     ///    - name: The name of the subdirectory
-    ///    - partitionIdentifier: The main paritition identifier to use
+    ///    - partitionIdentifier: The main partitionfetched remote config
+    ///     identifier to use
```

Comment:
Could it be that there was some kind of error here when you were typing?

Reply:
Ah yes thanks for catching that, did an incorrect find and replace!